model/project_test.go

```diff
@@ -1416,16 +1414,15 @@ func TestFindProjectsSuite(t *testing.T) {
 			},
 			{
 				Id:          "projectC",
-				Private:     utility.TruePtr(),
 				Enabled:     utility.TruePtr(),
 				CommitQueue: CommitQueueParams{Enabled: utility.TruePtr()},
 				Owner:       "mongodb",
 				Repo:        "mongo",
 				Branch:      "main",
 			},
-			{Id: "projectD", Private: utility.FalsePtr()},
-			{Id: "projectE", Private: utility.FalsePtr()},
-			{Id: "projectF", Private: utility.TruePtr()},
+			{Id: "projectD", Repo: evergreen.PublicRepoName},
+			{Id: "projectE", Repo: evergreen.PublicRepoName},
+			{Id: "projectF"},
```

Comment:
It seems like the case on L1420 would already be testing a private repo, so can we remove this project ref?